docs/source/AdministratorGuide/Tutorials/basicTutoSetup.rst

```diff
@@ -0,0 +1,765 @@
+      }
+    }
+    Hosts
+    {
+      dirac-tuto
+      {
+        DN = /C=ch/O=DIRAC/OU=DIRAC CI/CN=dirac-tuto/emailAddress=lhcb-dirac-ci@cern.ch
+        Properties = TrustedHost
+        Properties += CSAdministrator
+        Properties += JobAdministrator
+        Properties += FullDelegation
+        Properties += ProxyManagement
+        Properties += Operator
+      }
+    }
+    DefaultGroup = dirac_user
+  }
+  Operations
+  {
+    Defaults
+    {
+      EMail
+      {
+        Production = adminUser@cern.ch
+        Logging = adminUser@cern.ch
+      }
+    }
+  }
+  WebApp
+  {
+    Access
+    {
+      upload = TrustedHost
+    }
+  }
+  Systems
+  {
+    Framework
+    {
+      Production
+      {
+        Services
+        {
+          ComponentMonitoring
+          {
+            Port = 9190
+            Authorization
+            {
+              Default = ServiceAdministrator
+              componentExists = authenticated
+              getComponents = authenticated
+              hostExists = authenticated
+              getHosts = authenticated
+              installationExists = authenticated
+              getInstallations = authenticated
+              updateLog = Operator
+            }
+          }
+          SystemAdministrator
+          {
+            Port = 9162
+            Authorization
+            {
+              Default = ServiceAdministrator
+              storeHostInfo = Operator
+            }
+          }
+        }
+        URLs
+        {
+          ComponentMonitoring = dips://dirac-tuto:9190/Framework/ComponentMonitoring
+          SystemAdministrator = dips://dirac-tuto:9162/Framework/SystemAdministrator
+        }
+        FailoverURLs
+        {
+        }
+        Databases
+        {
+          InstalledComponentsDB
+          {
+            DBName = InstalledComponentsDB
+            Host = localhost
+            Port = 3306
+          }
+        }
+      }
+    }
+  }
+
+
+This configuration will be used for example by Services in order to:
+* know their configuration (for example the `ComponentMonitoring` Service will use everything under `Systems/Framework/Production/Services/ComponentMonitoring` )
+* Identify host and persons (`Registry` section)
+
+Or by clients to get the URLs of given services (for example `ComponentMonitoring = dips://dirac-tuto:9190/Framework/ComponentMonitoring`)
+
+Since this configuration is given as a whole to every client, you understand why no database credentials are in this file. Services and Agents running on the machine will have their configuration as a merge of what is served by the Configuration service and the `/opt/dirac/etc/dirac.cfg`, and thus have access to these private information.
+
+The file `/opt/dirac/bashrc` is to be sourced whenever you want to use the server installation.
+
+Client installation
+===================
+
+Now we will create another linux account `diracuser` and another installation to be used as client
```

Comment:
Maybe you mention that for simplicity this can be done on the same machine, but can also be done on a different one. It's pretty obvious, of course, but shows that this installation is fully capable.

Reply:
Since it is a tutorial, I'd rather be in favour of NOT giving any choice nor option. Otherwise you risk having people diverging. I can mention it, but I am not sure that it brings anything

Reply:
We need to decide if what you are writing here is either:

1. "the tutorial for the workshop", or 
2. "a general tutorial", that will also be used in the workshop to demonstrate something.

Of course you started with option 1, as this is the use case. My question is "can we make it a general one, so option 2"?

There are in the wiki some links to past tutorials/hands-on: https://github.com/DIRACGrid/DIRAC/wiki/DIRAC-Tutorials. There is also the one of last year's workshop. What we should aim to, in this "official documentation" is for general tutorials that readers can do alone. What you will do in the workshop can follow the lines outlined here, with maybe some deviations for convenience.

Reply:
What I am aiming at is option 2. And Tutorial at the DUW should just follow that. But to save time, this part would already be done for them

Reply:
I just went through the wiki. 
We duplicate so much the source of information :-)
How about just linkinf readthedocs from the wiki ?